app/assets/stylesheets/application.scss

```diff
@@ -18,3 +18,25 @@
 select {
   display: block;
 }
+
+h1 {
+    font-family: 'Playfair Display', serif;
+    font-weight: 900;
+    line-height: 46px;
+    font-size: 43px;
+    margin: 0 0 23px 0;
+    }
+h4 {
```

Comment:
I like that the paragraphs and headers are different fonts

Reply:
I chose this from a list of fonts that 'paired well together'

Reply:
With Cassie on this